app/core/voyagerUtils.js

```diff
@@ -0,0 +1,93 @@
+
+module.exports = {
+
+  /**
+   * Creates a level status map using the list of level session objects.
+   * @param {Object[]} sessions - The list of level session objects.
+   * @returns {Object} - Object with key as the level original id, and value as complete/started.
+   */
+  getLevelStatusMap: function (sessions) {
+    let levelStatusMap = {}
+    for (let session of sessions) {
```

Comment:
Nitpick: Similar comment about `const`. [Reference for my reasoning](https://medium.com/tanda-developers/why-do-we-prefer-const-d805b9dc03be)

Reply:
Thank you for adding that link, it was useful.